src/extension.ts

```diff
@@ -79,23 +79,68 @@ let decorationType: TextEditorDecorationType = window.createTextEditorDecoration
   }
 );
 
+const jdk8 = "1.8.232";
+const jdk8Action = "JDK8";
+
+const jdk11 = "1.11";
+const jdk11Action = "JDK11";
+
+const default_jabba_ver = "0.11.2";
```

Comment:
nit: please use camel case for identifier names.

```suggestion
const defaultJabbaVer = "0.11.2";
```

Reply:
Changed.